panc/pom.xml

```diff
@@ -133,6 +133,26 @@
         <executions>
 
           <execution>
+            <id>test-jar</id>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <configuration>
+              <archive>
+                <manifest>
+                  <mainClass>org.quattor.pan.PanTest</mainClass>
+                </manifest>
+              </archive>
+              <descriptors>
+                <descriptor>src/main/assembly/assembly.xml</descriptor>
```

Comment:
if this assembly file is only relevant for this fat jar, bets rename it to `assembly-fat.xml` or so